### DISCUSSION TITLE
install magpie in conda environment

### DISCUSSION THREAD

**Discussion by lvkas521424** [idaholab/moose#27675] [Q&A Getting Started]
### Check these boxes if you have followed the posting rules.

- [x] Q&A Getting Started is the most appropriate category for my question (trouble installing, beginner user, ...)
- [X] I have consulted the posting Guidelines on the Discussions front page
- [X] I have searched the Discussions forum and my question has not been asked before
- [X] I have searched the MOOSE website and the documentation does not answer my question
- [X] I have formatted my post following the posting guidelines (avoid screenshots if possible, triple back quotes before/after pasted text, etc)

### Issue or question about MOOSE

Hello, contributors
After configuring the moose environment in ubuntu, I followed magpie's instructions to install it, but had this problem

```
(moose) lvjiahui@lvkas:~/projects$ git clone https://github.com/idaholab/magpie.git
Cloning into 'magpie'...
remote: Enumerating objects: 10920, done.
remote: Counting objects: 100% (1853/1853), done.
remote: Compressing objects: 100% (1014/1014), done.
remote: Total 10920 (delta 850), reused 1812 (delta 836), pack-reused 9067
Receiving objects: 100% (10920/10920), 12.47 MiB | 1.04 MiB/s, done.
Resolving deltas: 100% (6084/6084), done.
(moose) lvjiahui@lvkas:~/projects$ git config --global http.postBuffer 4096M
(moose) lvjiahui@lvkas:~/projects/magpie$ git checkout master
branch 'master' set up to track 'origin/master'.
Switched to a new branch 'master'
(moose) lvjiahui@lvkas:~/projects/magpie$ git submodule update --init --recursive
Cloning into '/home/lvjiahui/projects/magpie/moose'...
fatal: unable to access 'https://github.com/idaholab/moose.git/': Could not resolve host: github.com
fatal: clone of 'https://github.com/idaholab/moose.git' into submodule path '/home/lvjiahui/projects/magpie/moose' failed
Failed to clone 'moose'. Retry scheduled
Cloning into '/home/lvjiahui/projects/magpie/moose'...
```

Is it just my network or is it not configured properly?

lvjiahui

**Comment by GiudGiud** [comment]
`git config --global http.postBuffer 4096M`

this is not in our instructions?

**Reply by lvkas521424** [reply]
yeah，I suspected that there was a problem with my network, so I tried to fix it with this command, but it didn't seem to do anything.

**Reply by GiudGiud** [reply]
does `git clone https://github.com/idaholab/moose.git` work outside of magpie?

**Reply by lvkas521424** [reply]
Sure, it's normal.

> (moose) lvjiahui@lvkas:~/projects$ git clone [https://github.com/idaholab/moose.git`](https://github.com/idaholab/moose.git%60)
> Cloning into 'moose'...
> remote: Enumerating objects: 752762, done.
> remote: Counting objects: 100% (3802/3802), done.
> remote: Compressing objects: 100% (1947/1947), done.
> Receiving objects: 3% (26559/752762), 7.41 MiB | 1001.00 KiB`

**Reply by GiudGiud** [reply]
ok. can you rm -rf moose in magpie/ and try again?

and also did you modify any of the .git files in magpie?

**Reply by lvkas521424** [reply]
Ok, I will try it tomorrow and I didn't modify anything.

It's time to sleep in my country. Thank you very much. Godbye and Good night.

**Reply by lvkas521424** [reply]
@GiudGiud Thanks sir, after deleting the moose folder, I re-downloaded and compiled the magpie. And now it's successful. 
Lvjiahui
